### ISSUE TITLE
Swallows warnings

### ISSUE THREAD

**Issue by Globegitter** [BadgeLabs/mocha-eslint#7]
I am just using this in one of my personal projects and I, as expected got an error for `"no-unused-vars": 2,` in my tests. Then I changed it to `"no-unused-vars": 1,` so the tests would pass, but in addition to that it is not even showing the warning.

My suspicion is, if it just has warnings to display this somehow gets swallowed. Because warnings seem to show fine when you have errors.

**Comment by IanVS** [comment]
@Globegitter This is how I intended it to behave.  The meaning of a "1" is that you would like to know about it, but you don't want it to cause your test to fail.  Mocha only has the concept of a test passing or failing.  I don't think it would be correct to cause an ESLint warning to fail a mocha test, thereby perhaps causing a continuous integration build to fail.  So, the compromise I arrived at is to only fail the mocha tests if there are ESLint errors, and if there are errors, then any warnings will also be displayed as well.
